tests/sanity/test/install/test-installed-files.js

```diff
@@ -31,6 +37,27 @@ describe('verify installed files', function() {
     await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/components/jobs-api/bin/jobs-api-server-*.jar`);
   });
 
+  it('fingerprint directory should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -d ${process.env.ZOWE_ROOT_DIR}/fingerprint`);
+  });
+
+  it('fingerprint RefRuntimeHash-*.txt should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/fingerprint/RefRuntimeHash-*.txt`);
+  });
+
+  // # verify the checksums of ROOT_DIR, to self-check zowe-verify-authenticity.sh
+  // $binDir/zowe-verify-authenticity.sh # No parameters! 
+  // You need to 'source' the profile to get JAVA_HOME
+
+  it('fingerprint should match', async function() {
+    await sshHelper.executeCommandWithNoError(`. ~/.profile && ${process.env.ZOWE_ROOT_DIR}/bin/zowe-verify-authenticity.sh | tee ~/zowe-verify-authenticity.txt`);
```

Comment:
Sorry I didn't realize you want to keep both (mocha test and verify playbook). If we decide to run `zowe-verify-authenticity.sh` here, then we don't need to also add it to `verify.yml` because verify playbook will run this whole set of mocha test. We can pick either one. Based on the current check, I would pick this test since it verifies more than what you added to verify playbook (like checking folders).

But few things of this line:

1. `~/.profile` may not exists, so `~/.profile` may fail on other servers.
2. `JAVA_HOME` is not a requirement defined here `tests/sanity/README.md`. If we run test from here, we will need to require JAVA_HOME to be set in `~/.profile`. I did added this requirement to `playbooks/README.md` and notified Kathy's team. This means anyone don't use playbook and try to run this sanity test directly, they may have issue.
3. I would recommend not to write anything to file system, like `~/zowe-verify-authenticity.txt`. You will need to clean it up, or you may have potential issues if the user home folder is full. Just write to stdout and check the result.

If we decide to pick verify playbook other than this mocha test, then it's better to put into `playbooks/roles/verify/tasks` folder as a dedicate task, and import from `playbooks/roles/verify/tasks/main.yml`. This way all fingerprints related tests will be in one file and easier to locate issues.

Reply:
Now only mocha is used.